jetson/science_bridge/src/__main__.py

```diff
@@ -249,22 +248,44 @@ def mosfet_transmit(self, channel, msg):
         pass
 
     def nav_status(self, channel, msg):  # TODO - fix make better
-        print("Received nav req")
         # Off, Done, Else
-
         struct = NavStatus.decode(msg)
+
+        if self.previous_auton_msg == struct.nav_state_name:
+            return
+
+        print("Received new nav req")
```

Comment:
Might be worth it to print out what the status name is. Though I guess we're gonna change all this anyways

Reply:
^ agree would be useful to print in the meantime

Reply:
Done